src/Workspaces/CSharp/Portable/Extensions/ITypeSymbolExtensions.TypeSyntaxGeneratorVisitor.cs

```diff
@@ -145,6 +147,22 @@ public TypeSyntax CreateSimpleTypeSyntax(INamedTypeSymbol symbol)
                     SyntaxFactory.TypeArgumentList(SyntaxFactory.SeparatedList(typeArguments)));
             }
 
+            private TupleTypeSyntax CreateTupleTypeSyntax(INamedTypeSymbol symbol)
+            {
+                var list = new SeparatedSyntaxList<TupleElementSyntax>();
+                var types = symbol.TupleElementTypes;
+                var names = symbol.TupleElementNames;
+                bool hasNames = !names.IsDefault;
```

Comment:
This check here seems to indicate the contract for `TupleElementNames` is to return `default(ImmutableArray<string>)` in the case of no names vs. `ImmutableArray<string>.Empty`.  We shuold make sure there is a test for this.